app/models/bgs_dependents/adult_child_attending_school.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class AdultChildAttendingSchool < Base
+    attribute :first, String
+    attribute :middle, String
+    attribute :last, String
+    attribute :suffix, String
+    attribute :ssn, String
+    attribute :birth_date, String
+    attribute :ever_married_ind, String
+
+    def initialize(dependents_application)
+      @dependents_application = dependents_application
+      @was_married = @dependents_application.dig('student_address_marriage_tuition', 'was_married')
```

Comment:
Why not make an attribute?

Reply:
I thought it would be cool to break this up in the initializer. We _sort_ of do make it an attribute. We use the boolean return value from this to create an attribute `BGS` expects. That attribute is `ever_married_ind` and it has to be 'Y' or 'N'